cv/pothole_detection/src/runonnx.py

```diff
@@ -0,0 +1,183 @@
+from math import ceil, floor
+import onnxruntime
+import numpy as np
+import time
+import cv2
+
+
+def xywh2xyxy(x):
+    # Convert nx4 boxes from [x, y, w, h] to [x1, y1, x2, y2] where xy1=top-left, xy2=bottom-right
+    y = np.copy(x)
+    y[:, 0] = x[:, 0] - x[:, 2] / 2  # top left x
+    y[:, 1] = x[:, 1] - x[:, 3] / 2  # top left y
+    y[:, 2] = x[:, 0] + x[:, 2] / 2  # bottom right x
+    y[:, 3] = x[:, 1] + x[:, 3] / 2  # bottom right y
+    return y
+
+
+def non_max_suppression(prediction, conf_thres=0.4, iou_thres=0.6):
+    """
+    prediction = [[[x, y, w, h, confidence, class], ...]]
+
+    returns an nx6 with (x1, y1, x2, y2, confidence, class)
+    """
+
+    candidates = prediction[..., 4] > conf_thres
+
+    max_det = 300
+    time_limit = 5.0
+
+    t = time.time()

```

Comment:
Remove benchmarking

Reply:
Going to leave but reduce interval. It's a good check to have to prevent this from reducing FPS for if we get lots of detections